pysph/base/gpu_nnps_helper.py

```diff
@@ -23,7 +23,7 @@ def get_elwise_kernel(kernel_name, args, src, preamble=""):
         ctx, args, src,
         kernel_name, preamble=preamble
     )
-    return profile_kernel(knl, kernel_name)
+    return profile_kernel(knl, kernel_name, backend='opencl')
```

Comment:
Why is this set to explicitly only the opencl backend?  Shouldn't this depend on the specific backend, what happens when CUDA is used?